content/03.use-cases-a.md

```diff
@@ -47,3 +47,10 @@ There are (as always) no "points" awarded for using ALL GitHub materials ALL the
 
 Matthew D. Beckman, Mine Çetinkaya-Rundel, Nicholas J. Horton, Colin W. Rundel, Adam J. Sullivan & Maria Tackett (2021) Implementing Version Control With Git and GitHub as a Learning Objective in Statistics and Data Science Courses, Journal of Statistics and Data Science Education, 29:sup1, S132-S144, DOI: 10.1080/10691898.2020.1848485
 
+
+Github is a valuable tool/platform to support both learning and teaching as a submission platform (to host their work and submit their course assignments and
+projects). Therefore Github can be used as a way to host course content, through basic GitHub, the organizations feature, and GitHub classroom.Therefore teachers can utilize GitHub as typical learning management systems to improve assignment submission, grading and feedback systems [@doi:10.2190/1002-1T50-27G2-H3V7].
```

Comment:
This seems like a good introduction to this section, not a conclusion paragraph.  Consider revising the beginning of the section instead.

Reply:
@Aariq I just removed "therefore". Does it make sense now?